packages/serve-cli/package.json

```diff
@@ -36,7 +36,7 @@
   "typings": "dist/typings/index.d.ts",
   "scripts": {
     "bundle": "(rm -rf bundle || true) && rollup -c",
-    "inject-version": "tsx scripts/inject-version.ts",
```

Comment:
Is this removed on purpose?

Reply:
https://github.com/ardatan/graphql-mesh/pull/7339/commits/54923a286f3799d9393cb2a38df8623aea3fe297